.env.example

```diff
@@ -0,0 +1,10 @@
+# DB_TYPE can be "sqlite", "mysql" or "postgres"
+DB_TYPE="postgres"
+# Only for sqlite:
+DB_STORAGE="db.sqlite"
+# Only for mysql and postgres:
+DB_HOST="localhost"
+DB_NAME="postgres"
```

Comment:
mmm actually postgres is the name of the default database created when you install postgres, this could cause conflicts, better choose something else like flugzeug-lib-test

Reply:
Got it